packages/@aws-cdk/aws-apigatewayv2-integrations/README.md

```diff
@@ -26,6 +26,37 @@
 
 Integrations connect a route to backend resources. HTTP APIs support Lambda proxy, AWS service, and HTTP proxy integrations. HTTP proxy integrations are also known as private integrations.
 
+### Amazon EventBridge
+
+You can integrate your HTTP API with AWS services by using first-class integrations. A first-class integration connects an HTTP API route to an AWS service API. When a client invokes a route that's backed by a first-class integration, API Gateway invokes an AWS service API for you.
+
+At this time only the EventBridge PutEvents integration is supported. Each AWS service integration subtype expects a set of `requestParamters` as part of the configuration. Detailed information with respect to the AWS service integration subtypes and request parameter formats can be found at [Integration Subtype Reference](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-aws-services.html).
```

Comment:
Change the language to match the title of this section. Provide 1-2 sentences on what this integration does and that `PutEvents` in the only supported API.

Also, let's keep line width to 120 characters.

Reply:
Can you clarify? Do you mean that the language should be specific to only EventBridge and not describe that this is an instance of an AWS Service Integration?

Reply:
Correct, yes. Talk about the event bridge notification in this section rather than AWS service integrations.

Reply:
Updated. Please reopen if I need to make it clearer. Thanks!